assets/settings/default.json

```diff
@@ -539,6 +539,12 @@
       // "delay_ms": 600
     }
   },
+  // Configuration for how direnv configuration should be loaded. May take 2 values:
+  // 1. Load direnv configuration through the shell hook
+  //      "direnv": "shell_hook"
+  // 2. Load direnv configuration using `direnv export json` directly
+  //      "direnv": "direct"
```

Comment:
Can you add an explanation for why you would want to use one over the other?

Reply:
And can you add this to `docs/src/configuring-zed.md`?

Reply:
Yeah, sure, it turned out a bit clumsy in my opinion, so if you have any tips on improving it I would appreciate that